Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -293,20 +292,18 @@
      * the class during application runtime.</p>
      */
     private static Branch branchReferral_;
-    
+
     private BranchRemoteInterface branchRemoteInterface_;
-    private final PrefHelper prefHelper_;
+    final PrefHelper prefHelper_;
```

Comment:
all this moving of methods and parameters in this class from `private` to package private is meant to accommodate moving out the BranchPostTask to its own file.